### ISSUE TITLE
Updated App Won't Launch

### ISSUE THREAD

**Issue by armstrong360** [standardnotes/forum#282]
I just downloaded the Mac desktop app update and now the app doesn’t work.  It relaunched automatically but no windows opened.  I quite it and restarted.  The icon bounced around but the app didn’t launch.  
I’m on a MacBook Pro running OS 10.12.6. 
What’s the solution?
Can I trash the app and reinstall? If so, how?
Thanks

**Comment by mobitar** [comment]
@armstrong360 I just replied to your email, but I'll post it here as well in case anyone else is having the same issue:

That’s strange. Can you try going back to 2.3.3 and see if the app works then? https://github.com/standardnotes/desktop/releases

If so, that would be surprising that the new update introduced some kind of issue. But also helpful info to know.

But yes, you would just trash the current app, and reinstall the downloaded file to your Applications folder.

**Comment by sycherown35epr** [comment]
Hi Mo,

I had the same problem (probably).

I logged out of all sessions and did the crypto update.  Then I tried to login to all my devices.  All authenticated fine and downloaded the data, except an iPhoneSE.  I deleted the app and re-downloaded it.  After the reinstall, the authentication worked fine.

This is only speculation.  It seemed to fail after I entered the username/password and the time-based-pseudorandom number. (But, after the reinstall from the Apple AppStore, the authentication worked fine.)

This is a great app.

-All the best to you
